examples/local_elasticsearch.conf

```diff
@@ -0,0 +1,26 @@
+input {
+  http_poller {
+    urls => {
```

Comment:
I think having rich options enables some very powerful monitoring use
cases.

We need header support to deal with Apis that are picky there. We need http
method support to allow people to use this as a health check ( can I create
an object? ).

That said I left the simple syntax in place because I think it covers a
major use case.

On Thursday, July 2, 2015, Jordan Sissel <notifications@github.com> wrote:

> In examples/local_elasticsearch.conf
> <https://github.com/logstash-plugins/logstash-input-http_poller/pull/17#discussion_r33800194>
> :
>
> > @@ -0,0 +1,26 @@
> > +input {
> > +  http_poller {
> > +    urls => {
>
> historically I have tried to avoid complex configuration settings like
> this. Worries I have - Is the complexity worth it? Can we document it well
> enough?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/logstash-plugins/logstash-input-http_poller/pull/17/files#r33800194>
> .
>